cartography/models/semgrep/findings.py

```diff
@@ -68,6 +68,40 @@ class SemgrepSCAFindingToGithubRepoRel(CartographyRelSchema):
     properties: SemgrepSCAFindingToGithubRepoRelProperties = SemgrepSCAFindingToGithubRepoRelProperties()
 
 
+@dataclass(frozen=True)
+class SemgrepSCAFindngToDependencyRelProperties(CartographyRelProperties):
+    lastupdated: PropertyRef = PropertyRef('lastupdated', set_in_kwargs=True)
+
+
+@dataclass(frozen=True)
+# (:SemgrepSCAFinding)-[:AFFECTS]->(:Dependency)
+class SemgrepSCAFindingToDependencyRel(CartographyRelSchema):
+    target_node_label: str = 'Dependency'
+    target_node_matcher: TargetNodeMatcher = make_target_node_matcher(
+        {'id': PropertyRef('matchedDependency')},
+    )
+    direction: LinkDirection = LinkDirection.OUTWARD
+    rel_label: str = "AFFECTS"
+    properties: SemgrepSCAFindngToDependencyRelProperties = SemgrepSCAFindngToDependencyRelProperties()
+
+
+@dataclass(frozen=True)
+class SemgrepSCAFindingToCVERelProperties(CartographyRelProperties):
+    lastupdated: PropertyRef = PropertyRef('lastupdated', set_in_kwargs=True)
+
+
+@dataclass(frozen=True)
+# (:SemgrepSCAFinding)<-[:LINKED_TO]-(:CVE)
+class SemgrepSCAFindingToCVERel(CartographyRelSchema):
+    target_node_label: str = 'CVE'
+    target_node_matcher: TargetNodeMatcher = make_target_node_matcher(
+        {'id': PropertyRef('cveId')},
```

Comment:
Is the property cveId or id?
https://github.com/lyft/cartography/blob/f3f237cb9625f26205ead1da971d5fb7cf3127b0/cartography/intel/cve/feed.py#L45

Reply:
It's `cveId`, since its taken from https://github.com/lyft/cartography/blob/297434ebb8f6398ce9e56fea2cab4ceb92bf06f6/cartography/intel/semgrep/findings.py#L125